pkgs/desktops/gnome/misc/gnome-epub-thumbnailer/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv
+, lib
+, fetchurl
+, meson
+, ninja
+, pkg-config
+, gnome
+, gdk-pixbuf
+, glib-networking
+, libarchive
+, libxml2
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-epub-thumbnailer";
+  version = "1.7";
+
+  src = fetchurl {
+    url = "mirror://gnome/sources/gnome-epub-thumbnailer/${lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
+    sha256 = "sha256-S7Ah++RCgNuY3xTBH6XkMgsWe4GpG9e6WGvqDE+il1I=";
+  };
+
+  nativeBuildInputs = [
+    libxml2
+    meson
+    ninja
+    pkg-config
+  ];
+
+  buildInputs = [
+    gdk-pixbuf
+    glib-networking
```

Comment:
What is glib-networking for here? Thumbnailers are sandboxed so they really should not access network.

Reply:
Its taken for this dependency list. This might have been a wrong dependency however. https://gitlab.gnome.org/GNOME/gnome-epub-thumbnailer/-/blob/24829a20630ee5a004f00b5fdd3b6830f593503f/meson.build#L4

Reply:
`gio` is found without this dependency. removed

Reply:
`gio-2.0` is part of `glib` package.